File: backend/src/main/java/com/example/SEPDrive/controller/fahrerDTO.java
package com.example.SEPDrive.controller;

import java.time.LocalDate;
import com.example.SEPDrive.model.carClass;

public record fahrerDTO(

        Integer id,
        String userName,
        String email,
        String fistName,
        String lastName,
        LocalDate dateOfBirth,
        byte[] profilePhoto,
        carClass carClass
) {}

File: frontend/Angular/src/app/models/fahrerDTO.ts
export interface FahrerDTO {

  profilePhoto?: FormData;
  userName?: string;
  firstName?: string;
  lastName?: string;
  email?: string;
  dateOfBirth?: Date;
  password?: string;
  carClass?: 'klein'| 'Medium' | 'Deluxe';
  dtype : 'Fahrer';

}
